_security/configuration/tls.md

```diff
@@ -77,6 +76,41 @@ Name | Description
 `plugins.security.ssl.http.truststore_password` | Truststore password. Default is `changeit`.
 
 
+## Separate client and server certificates for transport layer TLS
+
+By default, transport layer TLS certificates need to be configured as both client (`TLS Web Client Authentication`) and server (`TLS Web Server Authentication`) in the certificate's `Extended Key Usage` section, as they take responsibility as server and client in internal communication between nodes.
```

Comment:
```suggestion
By default, transport layer TLS certificates need to be configured as both client (`TLS Web Client Authentication`) and server (`TLS Web Server Authentication`) in the certificate's `Extended Key Usage` section, because the certificates are responsible for the server's and client's internal communication between nodes.
```

Reply:
@Naarcha-AWS would it be a better explanation to say instead of:

```
-because the certificates are responsible for the server's and client's internal communication between nodes.

+bacause the nodes using the TLS certificates take on the responsibility of serving and receiving the communication requests internally.
```